docs/usage/closed-prs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Closed PRs
+description: How Renovate handles closed PRs
+---
+
+# Ignoring closed PRs
```

Comment:
I'm not sure we want an entire new page just for this. @HonkingGoose do you have any suggestions for where this information could be placed?

Reply:
I agree that this feature does not "deserve" it's own page.

For now I think this can go in the FAQ, until I can think of a better place...